LICENSE

```diff
@@ -209,3 +209,29 @@ with separate copyright notices and license terms. Your use of those
 components are subject to the terms and conditions of the following
 licenses.
 
+---------------------------------------------------------------------------
+The MIT License (http://opensource.org/licenses/mit-license.html)
+---------------------------------------------------------------------------
+
+Apache Geode bundles the following files under the MIT license:
+
+  - cotire (https://github.com/sakra/cotire), Copyright (c) 2012-
```

Comment:
Can you explain why something like Cotire would be included in the LICENSE file? It is used only at build time. Is it because it will be shipped in source form with the source distribution?

Reply:
Exactly.  The LICENSE file needs to declare things that are bundled in the release (the source distribution).  If we choose to provide a convenience binary, there would be a second LICENSE file that also includes any bundled binary dependencies.

Reply:
Would it be preferable to change the source to fetch Cotire from GitHub when PCH is enabled at configure time?

Reply:
Either way is fine in this case.  If the bundled dependency were Category X (e.g. GPL) you would need take the optional download approach.